test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/section/DoSection.java

```diff
@@ -626,32 +628,49 @@ public String toString() {
         return result;
     }
 
+    private static boolean matchWithRange(String nodeVersionSting, List<VersionRange> acceptedVersionRanges) {
```

Comment:
Can we move this yaml test runner change to its own PR? It deserves to be reviewed independently, not rolled up into a test change.

Reply:
Sure, will do

Reply:
Spun off in https://github.com/elastic/elasticsearch/pull/100794